edit-post/components/plugin-sidebar/sidebar-layout.js

```diff
@@ -0,0 +1,31 @@
+import './style.scss';
+
+/**
+ * WordPress dependencies
+ */
+import { IconButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const SidebarLayout = ( { onClose, title, children } ) => {
+	return (
+		<div
+			className="edit-post-plugin-sidebar"
```

Comment:
https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming

Not entirely clear in the documentation, but this is not the default export of the `index.js`, so is subject to the same rule "for any descendent of the top-level element".